inst/templates/wercker.yml

```diff
@@ -0,0 +1,5 @@
+box: rocker/hadleyverse
```

Comment:
@jimhester: No, I haven't seen your post until now, but somehow you have found mine ;-) Thanks for the credit. -- Wercker steps are a clean solution, it's great you've found the time to implement them.